index.html

```diff
@@ -1,25 +1,103 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Awesome books</title>
-    <link rel="stylesheet" href="/src/style.css" />
-  </head>
-
-  <body>
-    <h1>Awesome books</h1>
-    <ul id="books"></ul>
-    <hr />
-    <h2>Add a new book</h2>
-    <form action="#" id="form">
-      <input type="text" name="title" id="title" placeholder="Title" />
-      <input type="text" name="author" id="author" placeholder="Author" />
-      <button onclick="myBooks.newBook()" id="add-button">Add book</button>
-    </form>
-
-    <!-- SCRIPTS -->
-    <script src="src/main.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Awesome books</title>
+  <link rel="stylesheet" href="/src/style.css" />
+  <script src="https://kit.fontawesome.com/78d4ca6841.js" crossorigin="anonymous"></script>
+  <script type="text/javascript" src="https://moment.github.io/luxon/global/luxon.min.js "></script>
+</head>
+
+<body>
+  <header>
+    <div>
+      <h1><a onclick="showAllBooks()" href="#" class="logo">Awesome books</a></h1>
+      <nav>
+        <a id="list-link" onclick="showAllBooks()" href="#" class="highlight">List</a>
+        <a id="add-new-link" onclick="showAddNew()" href="#">Add new</a>
+        <a id="contact-link" onclick="showContactUs()" href="#">Contact</a>
+      </nav>
+    </div>
+    <p id="display-date"></p>
+  </header>
+
+  <div class="container">
+    <section id="all-books">
+      <h2>All awesome books</h2>
+      <ul id="books"></ul>
+    </section>
+
+    <section id="add-new" class="hidden">
+      <h2>Add a new book</h2>
+      <form action="#" id="form">
+        <input type="text" name="title" id="title" placeholder="Title" />
+        <input type="text" name="author" id="author" placeholder="Author" />
+        <button onclick="myBooks.newBook()" id="add-button">Add book</button>
+      </form>
+    </section>
+
+    <section id="contact-us" class="hidden">
+      <h2 class="h2-contact">Contact information</h2>
+      <p>Do you have any questions or just want to say "hello"?</p>
+      <p>You can reach out to us!</p>
+      <h3>👩🏼‍💻 Vanessa Aoki</h3>
+
+      <ul>
+        <li>
+          GitHub:
+          <a href="https://github.com/VanessaAoki" target="_blank">
+            @VanessaAoki
```

Comment:
your design should follow this [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements)

Here's a screenshot of how your design looks like now

<img width="579" alt="Screenshot 2021-08-12 at 21 21 23" src="https://user-images.githubusercontent.com/34798898/129264516-d07978dd-e297-4d8f-8e3b-af1d26719f83.png">


PS:

I was surprised to see the above design, I had to checkout to different branches, I was thinking you might have made a mistake or so, but after several checking, I still couldn't find anything close to the screenshot of the design you provided in the readme file. I also saw that you're comparing the navigation branch to the development branch which makes me think I'm on the right branch, maybe you didn't push some updates or so. 

I must say this, I was really wow when I saw the screenshot of the design in the readme but on cloning the project and seeing the above, I was not so happy

Please I don't know what might have gone wrong from your end but please update this and resubmit for review 😄

Reply:
Hello @abruzy,

First of all, thanks so much for the review!
Turned out we just had an additional "/" when linking our stylesheet, so our CSS was not linked at all.
It should be all fixed now!
If you prefer, here is the [live demo link](https://rawcdn.githack.com/VanessaAoki/AwesomeBooks/3ca746caa2332a3bc69bb2846545a09298e8045f/index.html).

Reply:
Oppps! Okay, great work fixing that 🚀 

I really love your design 😄 

Submit again for re-review